packages/yii-dev-panel/README.md

```diff
@@ -152,6 +152,32 @@ npm start
 
 `yii-dev-panel` is now serving on http://localhost:3000
 
-## Screenshot
-
-![Screenshot](docs/screenshot.png)
+## Screenshots
+
+
+<details>
```

Comment:
Do we need to hide these under spoilers?

Reply:
There are lots of images, I'd hide them to make page simple